src/com/example/hotel/Guest.java

```diff
@@ -0,0 +1,59 @@
+package com.example.hotel;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class Guest {
+    private String name;
```

Comment:
add please Id (unique identifier)for entities that you want to store, the easiest way it make it Long type